LocalPackages/VPNiOS/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 6.0
```

Comment:
Noting here that we're using Swift 6; I think this is fine in this case, but we should probably align as a team on how we want to approach this in the new year.

Reply:
Got it.  Marking as resolved as there's no change required.